packages/clay-loading-indicator/src/index.tsx

```diff
@@ -6,7 +6,26 @@
 import classNames from 'classnames';
 import React from 'react';
 
+type DisplayType = null | 'primary' | 'secondary' | 'light';
+type Shape = null | 'circle' | 'squares';
+type Size = null | 'xs' | 'sm' | 'md' | 'lg';
+
 interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Determines the color of the visual indicator.
+	 */
+	displayType?: DisplayType;
+
+	/**
+	 * Determines the style of the visual indicator.
+	 */
+	shape?: Shape;
+
+	/**
+	 * Determines the size of the visual indicator.
+	 */
+	size?: Size;
+
 	/**
 	 * Flag to indicate the 'light' variant
```

Comment:
For these two properties, we can deprecate since we can do the same now with `size` and `displayType`, you can add a comment in the descriptions of `light` and `small` like:

```js
// @deprecated since v3.59.0 - use `size` instead.
```